diff -p -up wrapitk-0.3.0/Languages/SwigInterface/CMakeLists.txt.orig wrapitk-0.3.0/Languages/SwigInterface/CMakeLists.txt
--- wrapitk-0.3.0/Languages/SwigInterface/CMakeLists.txt.orig	2010-11-04 22:00:49.100546043 -0200
+++ wrapitk-0.3.0/Languages/SwigInterface/CMakeLists.txt	2010-11-04 22:01:44.699545583 -0200
@@ -324,12 +324,12 @@ MACRO(ADD_ONE_TYPEDEF_SWIG_INTERFACE wra
 
   IF("${wrap_method}" MATCHES "2_SUPERCLASSES")
     ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Superclass::Superclass::Self" "${swig_name}_Superclass_Superclass")
-    ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Superclass::Superclass::Pointer::SmartPointer" "${swig_name}_Superclass_Superclass_Pointer")
+    ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Superclass::Superclass::Pointer" "${swig_name}_Superclass_Superclass_Pointer")
   ENDIF("${wrap_method}" MATCHES "2_SUPERCLASSES")
 
   IF("${wrap_method}" MATCHES "SUPERCLASS")
     ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Superclass::Self" "${swig_name}_Superclass")
-    ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Superclass::Pointer::SmartPointer" "${swig_name}_Superclass_Pointer")
+    ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Superclass::Pointer" "${swig_name}_Superclass_Pointer")
   ENDIF("${wrap_method}" MATCHES "SUPERCLASS")
 
   IF("${wrap_method}" MATCHES "FORCE_INSTANTIATE")
@@ -341,7 +341,7 @@ MACRO(ADD_ONE_TYPEDEF_SWIG_INTERFACE wra
 
   IF("${wrap_method}" MATCHES "POINTER")
     # add a pointer typedef if we are so asked
-    ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Pointer::SmartPointer" "${swig_name}_Pointer")
+    ADD_SIMPLE_TYPEDEF_SWIG_INTERFACE("${full_class_name}::Pointer" "${swig_name}_Pointer")
   ENDIF("${wrap_method}" MATCHES "POINTER")
 
 ENDMACRO(ADD_ONE_TYPEDEF_SWIG_INTERFACE)
